.gitmodules

```diff
@@ -90,3 +90,6 @@
 [submodule "examples_tests"]
 	path = examples_tests
 	url = git@github.com:Devsh-Graphics-Programming/Nabla-Examples-and-Tests.git
+[submodule "3rdparty/dxc/dxc"]
+	path = 3rdparty/dxc/dxc
+	url = git@github.com:Devsh-Graphics-Programming/DirectXShaderCompiler.git
```

Comment:
DX already provides SPIRV-Headeaders etc, can you boot these submodules?

Reply:
it's a TODO for the PR, yes

Reply:
resolved